installing/overview/cluster-capabilities.adoc

```diff
@@ -13,10 +13,15 @@ Cluster administrators can use cluster capabilities to enable or disable optiona
 Cluster administrators cannot disable a cluster capability after it is enabled.
 ====
 
-include::modules/selecting-cluster-capabilities.adoc[leveloffset=+1]
-
+include::modules/enabling-cluster-capabilities.adoc[leveloffset=+1]
 include::snippets/capabilities-table.adoc[]
 
+include::modules/viewing-cluster-capabilities.adoc[leveloffset=+1]
+
+include::modules/enabling-baseline-capability-set.adoc[leveloffset=+1]
+
+include::modules/enabling-additional-enabled-capabilities.adoc[leveloffset=+1]
```

Comment:
I think these modules should go at the bottom, after the list of Operators, since you can't engage with any of the "post-install" procedures until after you've gone through an install. WDYT?

Reply:
wfm!